src/libraries/System.Private.CoreLib/src/System/Threading/Timer.cs

```diff
@@ -444,6 +493,11 @@ internal TimerQueueTimer(TimerCallback timerCallback, object? state, uint dueTim
                 Change(dueTime, period);
         }
 
+        internal string DisplayString =>
+            "Due = " + (_dueTime == Timeout.UnsignedInfinite ? "(not set)" : (DateTime.Now + TimeSpan.FromMilliseconds(_dueTime - (TimerQueue.TickCount64 - _startTicks)))) + ", " +
+            "Period = " + (_period == Timeout.UnsignedInfinite ? "(not set)" : TimeSpan.FromMilliseconds(_period).ToString("g")) + ", " +
+            _timerCallback.Method;
```

Comment:
What does this look like for Cancellation Tokens and Task.Delay?

Reply:
![image](https://user-images.githubusercontent.com/2642209/110132819-92b90780-7d99-11eb-9248-4825bb3e1077.png)